openupgradelib/openupgrade.py

```diff
@@ -2195,12 +2195,18 @@ def convert_field_to_html(cr, table, field_name, html_field_name):
         }
     )
     for row in cr.fetchall():
-        logged_query(
-            cr, "UPDATE %(table)s SET %(field)s = %%s WHERE id = %%s" % {
-                'field': html_field_name,
-                'table': table,
-            }, (plaintext2html(row[1]), row[0])
-        )
+        query = "UPDATE %(table)s SET %(field)s = %%s WHERE id = %%s" % {
+            'field': html_field_name,
+            'table': table,
+        }
+        if verbose:
+            logged_query(
+                cr, query, (plaintext2html(row[1]), row[0])
+            )
```

Comment:
Everything fits in one line:

```suggestion
            logged_query(cr, query, (plaintext2html(row[1]), row[0]))
```